src/Microsoft.Health.Fhir.Api/Resources.resx

```diff
@@ -258,7 +258,18 @@
     <comment>{NumberedPlaceHolder=”STU3”}</comment>
   </data>
   <data name="InvalidBundleEntry" xml:space="preserve">
-    <value>Requested operation does not support {0} interaction using {1}.</value>
-    <comment>{0} is the interaction route and {1} is the http verb.</comment>
+    <value>Requested operation '{0}' is not supported using {1}.</value>
+    <comment>{0} is the operation and {1} is the http verb.</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="InvalidConditionalReference" xml:space="preserve">
+    <value>Given conditional reference '{0}' does not resolve to a resource.</value>
+    <comment>{0} is the conditional reference parameter.</comment>
+  </data>
+  <data name="InvalidConditionalReferenceParameters" xml:space="preserve">
+    <value>Resource type and query parameter must be present in a given request '{0}'.</value>
+    <comment>{0} is the request url.</comment>
+  </data>
+  <data name="ConditionalOperationNotSelectiveEnough" xml:space="preserve">
```

Comment:
Does something similar exist in `Microsoft.Health.Fhir.Core.Resources`?

Reply:
Yes but I was unable to refer that. Therefore added the same in API.